src/core/server/status/status_service.test.ts

```diff
@@ -535,5 +543,50 @@ describe('StatusService', () => {
         );
       });
     });
+
+    describe('analytics', () => {
+      let analyticsMock: jest.Mocked<AnalyticsServiceSetup>;
+      let setup: InternalStatusServiceSetup;
+
+      beforeEach(async () => {
+        analyticsMock = analyticsServiceMock.createAnalyticsServiceSetup();
```

Comment:
NIT do we really need to redefine this? Looks like we are initializing it with the same mock in the setupDeps().

Reply:
Yes... the `setupDeps` _helper_ claims it returns the final types, so you can't do

```ts
const setupContract = setupDeps();
setupContract.analytics.mockImplementation(...)
                                        ^ complains here because analytics is not a mock
```